### PR TITLE
fix: correct misspelled icons

### PR DESCRIPTION
**Pull request by epetrow**
closes: https://github.com/telerik/kendo-icons/issues/148

### CODE REVIEW

File: src/telerik-icons/icons.json [hunk 1]
```diff
@@ -5591,32 +5603,6 @@
...
-    {
-        "name": "envelop-box",
-        "ligatures": [],
-        "aliases": [
-            "email-box",
-            "letter-box"
-        ],
-        "categories": [
-            "social"
-        ],
```
Comment (Juveniel) [inline src/telerik-icons/icons.json:5616]: aliases and categories should also be t ransferred

**Review by radkostanev — APPROVED** [review]
The important bit in terms of the NuGet enumeration is that the unique Id remains the same(e.g. an icon that has been updated/renamed is still the same icon, thus the Id should persist). 

However, the fact that the name of the icon itself changes still might break usages relying on the name itself, but that is to be expected given the typo.

**Review by Juveniel — APPROVED** [review]

**Comment by Juveniel** [comment]
The main usage for the icon that I have found is in the Kendo UI for Angular examples ( [list](https://github.com/search?q=org%3Atelerik+envelopIcon&type=code) ). Thus, when the `kendo-svg-icons` packages is updated, we need to make sure the imports are also updated.

// cc @stnikolova @Raisolution

**Comment by tsvetomir** [comment]
That's a breaking change and the major version **must** be bumped to avoid breaking existing apps. Alternatively, keep an alias for the old icon name.

**Pull request merged**